classes/emlinux-compact.bbclass

```diff
@@ -29,6 +29,13 @@ def create_replace_script(d):
     user_specified_install_pkgs = image_preinstall + image_install
     pinned_pkgs = []
 
+    use_systemd = d.getVar("EMLINUX_IMAGE_COMPACT_USE_SYSTEMD", "0")
```

Comment:
Do you mean False instead of "0"?

Reply:
EMLINUX_IMAGE_COMPACT_USE_SYSTEMD takes "1" to enable this feature, so get "0" if this variable is not defined. If this variable is not 1, it will be ignored on the following process.

Reply:
The second argument to d.getVar() is a boolean or nothing.

I confirmed you fixed the above line. Thank you.